contracts/Vault.sol

```diff
@@ -607,17 +608,12 @@ contract Vault is
             );
             locals.accumulatedPct += data.pct;
             locals.accumulatedAmount += localAmount;
-            result[i] = id;
         }
 
         require(
             locals.accumulatedPct.is100Perc(),
             "Vault: claims don't add up to 100%"
         );
-        require(
```

Comment:
Is this no longer relevant?

Reply:
Yeah, it is always true because of the following line.

https://github.com/sandclock-org/solidity_contracts/pull/47/files/42fd773c53e53145f8193c80364244a9c659266f#diff-b679e4afcd7c2875a8285c48f7821134c7c8c8092faa9615fafb4c26b31a98ccR597

Reply:
I can see which line you're mentioning :/